### ISSUE TITLE
Can't make oxlint work

### ISSUE THREAD

**Issue by dpecos** [mfussenegger/nvim-lint#595]
I'm using oxlint as explained in their website:

```
npx oxlint@latest
```

and I get a bunch of findings.

But I can't make those diagnostics appear in neovim. `nvim-lint` does show diagnostics for other filetypes.

I've also install oxlint via `mason`, but nothing changed.

Any tips?

**Comment by jasonleelunn** [comment]
I had the same issue and after managing to log out the error from inside `nvim-lint`, it turns out that `mason` is installing an old version of `oxlint` (0.0.20) which doesn’t understand the “--format unix” arg which `nvim-lint` passes it

the latest `oxlint` version is 0.5.2, I have submitted a PR (https://github.com/mason-org/mason-registry/pull/6380) to bump the version in the mason registry which will fix this and make oxlint diagnostics appear correctly
